core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -142,6 +149,7 @@ private TableProperties() {
   public static final String WRITE_AUDIT_PUBLISH_ENABLED_DEFAULT = "false";
 
   public static final String WRITE_TARGET_FILE_SIZE_BYTES = "write.target-file-size-bytes";
+  public static final String DELETE_TARGET_FILE_SIZE_BYTES = "write.delete.target-file-size-bytes";
```

Comment:
So , are we planning to adjust the delete writer's target-file-size-bytes in the following PR ?  I see this property is referenced by nobody.

Reply:
I have a number of follow-up PRs. Just want to keep the scope of this change small.